src/mgr/MDSPerfMetricCollector.cc

```diff
@@ -47,6 +47,7 @@ int MDSPerfMetricCollector::get_counters(PerfCollector *collector) {
 
   get_delayed_ranks(&c->delayed_ranks);
 
+  c->last_updated = ceph_clock_now();
```

Comment:
Why here? This need to be updated in `MDSPerfMetricCollector::process_reports()`